docs/content/en/docs/concepts/metrics/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Metrics
+description: Learn what Keptn Metrics are and how to use them
+icon: concepts
+layout: quickstart
+weight: 10
+hidechildren: true # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+### Keptn Metric
+A `KeptnMetric` is a CRD used to define SLI provider with a query and to store metric data fetched from the provider. Providing the metrics as CRD into a K8s cluster will facilitate the reusability of this data across multiple components. Furthermore, this allows using multiple observability platforms for different metrics.
```

Comment:
I would rephrase the first part as:  ... is a CRD representing a metric. The metric will be collected from the provider specified in the specs.provider.name field. The query is a string containing provider specific query to obtain a metric. MAybe prometheus example also.

Reply:
real example is in readme, the other things I agree, can you make a suggestion what exactly you meant to change ?

Reply:
no I ment with a proper prometheus query in it

Reply:
ok we can add a query there, but maybe I will add it rather to readme than here